kiota_abstractions/_version.py

```diff
@@ -1 +1 @@
-VERSION: str = '0.2.2'
+VERSION: str = '0.2.3'
```

Comment:
```suggestion
VERSION: str = '0.3.0'
```
Since it's a backwards incompatible change during development?

Reply:
Thanks. That makes sense.